libs/orchestrator-ui-components/src/lib/components/OrchestratorPageTemplate/OrchestratorPageTemplate.tsx

```diff
@@ -68,7 +73,40 @@ export const OrchestratorPageTemplate: FC<OrchestratorPageTemplateProps> = ({
                     <EuiPageTemplate.Sidebar
                         css={{ backgroundColor: euiTheme.colors.body }}
                     >
-                        <div>Sidebar</div>
+                        <EuiSideNav
+                            mobileTitle="Nav Items"
+                            isOpenOnMobile={false}
+                            items={[
+                                {
+                                    name: 'Menu',
+                                    id: 1,
+                                    items: [
+                                        {
+                                            name: 'Home',
+                                            id: 2,
+                                            href: '/',
+                                            // TODO: NEXT router / EUI seem to cause unneeded re-renders. At least in dev mode,
```

Comment:
Can you try this one: https://beta.nextjs.org/docs/routing/linking-and-navigating#userouter-hook
You are importing from 'next/router', try to import from 'next/navigation'